pkg/bootstrap/bootstrap.go

```diff
@@ -321,21 +323,37 @@ func rewriteChart(fileName string, info os.FileInfo, chartValues map[string]stri
 		chart.Spec.Set = map[string]intstr.IntOrString{}
 	}
 
+	var changed bool
 	for k, v := range chartValues {
-		chart.Spec.Set[k] = intstr.FromString(v)
+		val := intstr.FromString(v)
+		if cur, ok := chart.Spec.Set[k]; ok {
+			curBytes, _ := cur.MarshalJSON()
```

Comment:
why do we ignore the error check here?

Reply:
The new value is guaranteed to be valid since we just generated it, and if the current value is invalid and returns an error we'll handle comparing the empty byte slice to the new value properly anyway despite the error.

Reply:
makes sense